ethstore/src/json/bytes.rs

```diff
@@ -19,7 +19,7 @@ use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use serde::de::Error;
 use rustc_hex::{ToHex, FromHex, FromHexError};
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Clone)]
```

Comment:
why did you add `derive(Clone)`?

Reply:
its probably aftermath my debugging... sorry